standalone/src/recording/components/reaction_stream.tsx

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Is the need to duplicate this here purely due to the issue with `<Provider>`?

Reply:
yep, only because of `<Provider>`.

Reply:
Out of curiosity, could you try adding this:

```patch
diff --git a/standalone/webpack.config.js b/standalone/webpack.config.js
index bb603c7..ff181f7 100644
--- a/standalone/webpack.config.js
+++ b/standalone/webpack.config.js
@@ -91,6 +91,7 @@ module.exports = {
             'react-dom': path.resolve(__dirname, './node_modules/react-dom'),
             'react-bootstrap': path.resolve(__dirname, './node_modules/react-bootstrap'),
             bootstrap: path.resolve(__dirname, './node_modules/bootstrap'),
+            'react-redux': path.resolve(__dirname, './node_modules/react-redux'),
         },
         fallback: {
             src: path.resolve(__dirname, '../webapp/src'),
```

And import from the webapp side instead.

Reply:
Ok, that works... But why? What does resolving react-redux have to do with importing from a different directory?

Reply:
No idea, I just found https://github.com/reduxjs/react-redux/issues/1489 and thought it was worth a try.

Reply:
What a find, thank you. That clarifies (at least a little) what was going wrong.